<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="wizardPageStart">
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>End of day upload</string>
   </property>
   <property name="subTitle">
    <string>This uploads new surveys and photos to the reefcloud hardware. You will be asked to add metadata to the surveys before the upload process begins.</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="wizardPageSurveysBefore">
   <property name="title">
    <string>Add survey data</string>
   </property>
   <property name="subTitle">
    <string>Add the survey data before uploading the images. If you skip this step you can do it later.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tblSurveysBefore"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPageUploadFinished">
   <property name="title">
    <string>Upload is finished</string>
   </property>
   <property name="subTitle">
    <string>next: review surveys</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="wizardPageSurveysAfter">
   <property name="title">
    <string>All surveys</string>
   </property>
   <property name="subTitle">
    <string>These are all the surveys currently stored on this computer</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTableView" name="tblSurveysAfter"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
